plugins/analytics-module-ga/README.md

```diff
@@ -0,0 +1,144 @@
+be sent to GA as your users navigate and use your Backstage instance.
+
+This plugin contains no other functionality.
+
+## Installation
+
+1. Install this plugin in your Backstage app:
+   `cd packages/app && yarn add @backstage/plugin-analytics-module-ga`
+2. Add the API implementation to your App and configure the plugin (see below).
+
+```tsx
+// packages/app/src/apis.ts
+import { analyticsApiRef, configApiRef } from '@backstage/core-plugin-api';
+import { GoogleAnalytics } from '@backstage/plugin-analytics-module-ga';
+
+export const apis: AnyApiFactory[] = [
+  // Instantiate and register the GA Analytics API Implementation.
+  createApiFactory({
+    api: analyticsApiRef,
+    deps: { configApi: configApiRef },
+    factory: ({ configApi }) => GoogleAnalytics.fromConfig(configApi),
+  }),
+];
+```
+
+## Configuration
+
+The following is the minimum configuration required to start sending analytics
+events to GA. All that's needed is your Universal Analytics tracking ID:
+
+```yaml
+# app-config.yaml
+app:
+  analytics:
+    provider: ga
+    ga:
+      trackingId: UA-0000000-0
+```
+
+### Plugin Analytics
+
+In order to be able to analyze usage of your Backstage instance _by plugin_, we
+strongly recommend configuring at least one custom dimension to capture Plugin
+IDs associated with events, including page views.
+
+1. First, [configure the custom dimension in GA][configure-custom-dimension].
+   Be sure to set the Scope to `hit`, and name it something like `Plugin`. Note
+   the index of the dimension you just created (e.g. `1`, if this is the first
+   custom dimension you've created in your GA property).
+2. Then, add a mapping to your `app.analytics.ga` configuration that instructs
+   the plugin to capture Plugin IDs on the custom dimension you just created.
+   It should look like this:
+
+```yaml
+app:
+  analytics:
+    provider: ga
+    ga:
+      trackingId: UA-0000000-0
+      customDimensionsMetrics:
+        - type: dimension
+          index: 1
+          source: context
+          key: pluginId
+```
+
+You can configure additional custom dimension and metric collection by adding
+more entries to the `customDimensionsMetrics` array:
+
+```yaml
+app:
+  analytics:
+    ga:
+      customDimensionsMetrics:
+        - type: dimension
+          index: 1
+          source: context
+          key: pluginId
+        - type: dimension
+          index: 2
+          source: context
+          key: routeRef
+        - type: metric
+          index: 1
+          source: attributes
+          key: someEventContextAttr
+```
+
+### Debugging and Testing
+
+In pre-production environments, you may wish to set additional configurations
+to turn off reporting to Analytics and/or print debug statements to the
+console. You can do so like this:
+
+```yaml
+app:
+  analytics:
+    ga:
+      testMode: true # Prevents data being sent to GA
+      debug: true # Logs analytics event to the web console
```

Comment:
Here do you mean to have an `app-config.local.yaml` containing this configuration?
If so, would it be good to mention the configuration split or not?